watchtower-plugin/src/retrier.rs

```diff
@@ -46,6 +52,15 @@ impl RetryError {
         )
     }
 }
+#[cfg(not(feature = "accountable"))]
```

Comment:
No, this can be done in a single function. Add the two common, and then conditionally add `Misbehaving` if accountable.

Reply:
I am not able to do this, features do not work in this way